site/content/en/latest/tasks/observability/proxy-metric.md

```diff
@@ -2,46 +2,128 @@
 title: "Proxy Metrics"
 ---
 
-Envoy Gateway provides observability for the ControlPlane and the underlying EnvoyProxy instances.
-This task show you how to config proxy metrics.
+Envoy Gateway offers observability for both the Control Plane and the underlying Envoy Proxy instances.
+This task shows you how to configure proxy metrics.
 
 ## Prerequisites
 
 {{< boilerplate o11y_prerequisites >}}
 
 ## Metrics
 
-By default, Envoy Gateway expose metrics with prometheus endpoint. 
+### Prometheus Metrics
 
-Verify metrics:
+To query metrics using Prometheus API, follow the steps below.
+
+```shell
+export PROMETHEUS_PORT=$(kubectl get service prometheus -n monitoring -o jsonpath='{.spec.ports[0].port}')
+kubectl port-forward service/prometheus -n monitoring 19001:$PROMETHEUS_PORT
+```
+
+Query metrics using Prometheus API:
+
+```shell
+curl -s 'http://localhost:19001/api/v1/query?query=topk(1,envoy_cluster_upstream_cx_connect_ms_sum)' | jq . 
+```
+
+To directly view the metrics in Prometheus format from the Envoy's `/stats/prometheus` 
+[admin endpoint](https://www.envoyproxy.io/docs/envoy/latest/operations/admin), follow the steps below.
 
 ```shell
 export ENVOY_POD_NAME=$(kubectl get pod -n envoy-gateway-system --selector=gateway.envoyproxy.io/owning-gateway-namespace=default,gateway.envoyproxy.io/owning-gateway-name=eg -o jsonpath='{.items[0].metadata.name}')
 kubectl port-forward pod/$ENVOY_POD_NAME -n envoy-gateway-system 19001:19001
+```
 
-# check metrics 
+View the metrics:
+
+```shell
 curl localhost:19001/stats/prometheus  | grep "default/backend/rule/0"
 ```
 
-You can disable metrics by setting the `telemetry.metrics.prometheus.disable` to `true` in the `EnvoyProxy` CRD.
+If you are only using the OpenTelemetry sink, you might want to set the `telemetry.metrics.prometheus.disable` to `true`
+in the _EnvoyProxy CRD_ as shown in the following command.
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
```

Comment:
can we link this to the `Gateway` instead of `GatewayClass` since thats a more common approach